src/AMDGPU.jl

```diff
@@ -153,6 +154,7 @@ function __init__()
                 Please run Pkg.build("AMDGPU") and reload AMDGPU.
                 """
             end
+            active_kernels[get_default_queue()] = Vector{AMDGPU.RuntimeEvent{AMDGPU.HSAStatusSignal}}() 
```

Comment:
This should move to the `HSAQueue` constructor, which will be invoked by `get_default_queue()` on its first call for a given agent.